cypress/integration/pages/testsForAllPages.js

```diff
@@ -19,12 +19,15 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
   describe(`Running testsForAllPages for ${service} ${pageType}`, () => {
     describe(`Metadata`, () => {
       it('should have resource hints', () => {
-        const resources = [
-          envConfig.assetOrigin,
-          'https://ichef.bbci.co.uk',
-          'https://gel.files.bbci.co.uk',
-          'https://ws-downloads.files.bbci.co.uk',
-        ];
+        const resources = [envConfig.assetOrigin, 'https://ichef.bbci.co.uk'];
+        const serviceConfig = appConfig[service];
+        const { fonts } = serviceConfig[variant || 'default'];
```

Comment:
I think the default value for `variant` should be set in the function signature.